plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -370,6 +370,50 @@ fu_plugin_uefi_capsule_update_splash (FuPlugin *plugin, FuDevice *device, GError
 	return fu_plugin_uefi_capsule_write_splash_data (plugin, device, image_bmp, error);
 }
 
+static gboolean
+fu_capsule_update_on_disk (FuDevice *device, GError **error)
+{
+	g_autofree guint64 *os_indications_supported = NULL;
+	g_autofree guint64 *os_indications = NULL;
+	guint32 attr;
+	gsize os_ind_size = 0;
+
+	if (fu_device_get_metadata_boolean (device, "DisableCapsuleUpdateOnDisk"))
+		return FALSE;
+
+	/* check for capsuleupdate ondisk */
+	if (!fu_efivar_get_data (FU_EFIVAR_GUID_EFI_GLOBAL, "OsIndicationsSupported",
+				 (guint8 **) &os_indications_supported,
+				 &os_ind_size, &attr, error))
+		g_debug ("Can't Read OsIndicationsSupported");
```

Comment:
you'll need to return false here and prefix error

Reply:
returning false already is the case. Effectively the next if won't succeed,, so you'll end up at the bottom. What do you mean by prefix the error?